### ISSUE TITLE
`hide_when_no_warning` hides all cards even when only one is hidden

### ISSUE THREAD

**Issue by RubenKelevra** [MrBartusek/MeteoalarmCard#68]
**Describe the bug**

I have the following view configured:

```
title: Rooms
views:
  - title: Outdoor
    path: outdoor
    cards:
      - type: custom:meteoalarm-card
        entity: sensor.dwd_weather_warnings_current_warning_level
        hide_when_no_warning: true
        integration: dwd
      - type: custom:meteoalarm-card
        entity: sensor.dwd_weather_warnings_advance_warning_level
        hide_when_no_warning: true
        integration: dwd
```

If the advance_warning is empty, the current warning won't be shown as well. Even if I have `hide_when_no_warning: false` on the current warning.

**Expected behavior**
Only the empty warning display should be removed OR they should only be removed if both are empty.

**System (please complete the following information):**
 - Card version: No idea where to find this info.
 - Integration type: DWD
 - Installation type: HACS

**Comment by MrBartusek** [comment]
> Card version: No idea where to find this info.

card version can be found in the developer console

![image](https://user-images.githubusercontent.com/23432278/151802574-c09d2111-4f7b-4e6a-9757-5424b6949582.png)

**Comment by MrBartusek** [comment]
This fix is now part of the new beta version [`v2.0.0-rc.2`](https://github.com/MrBartusek/MeteoalarmCard/releases/tag/v2.0.0-rc.2). You can help to test changes in the new release. See more here: #88
